modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,93 @@
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create a user workloads in Local Zone subnets. 
+
+After the `openshift-installer` creates the cluster, the installer automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod to a node if the pod's deployment does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+
+The `openshift-installer` creates the Machine Set manifest and attaches `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels to each edge worker node that is located in a Local Zone subnet.
+
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You configured an AWS Local Zones environment. 
+* You deployed your cluster om a compatible cloud platform. 
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations for your edge worker node, so that the scheduler can schedule any pods with matching taints to the cluster.
++
+.Example of a configured portion of the worker manifest file that the VPC uses to run workloads
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment <1>
+metadata:
+  name: <local_zone_name> <2>
+  namespace: <local_zone_namespace> <3>
+spec:
+  selector: 
+    matchLabels:
+      app: <local_zone_application>
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        app: <local_zone_application>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME} <4>
+    spec:
+      nodeSelector:
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME}
+      tolerations: <5>
+      - key: "node-role.kubernetes.io/edge"
+        operator: "Equal"
+        value: ""
+        effect: "NoSchedule"
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: echoserver
+          ports:
+            - containerPort: 8080
+----
+<1> `kind`: Defines the `deployment` resource.
+<2> `name`: Specifies the name of the service. 
+<3> `namespace:` Defines the namespace that the `service` resource targets.
+<4> `machine.openshift.io/zone-group`: Defines the group to where a zone belongs. For example, `us-east-1-nyc-1` indicates the region code, `nyc`, and identifiers for a Local Zone.
+<5> `tolerations`: Sets the tag where the scheduler can apply a taint to affected nodes.
+
+
+. Create a `service` resource YAML file for the node.
```

Comment:
@mtulio . What is the overall purpose of the `service` resource. After its creation does some mechanism, such as an Operator, use it?

Reply:
Service will expose the pod to the network. So basically is the way to expose one App/API to the other services running on the cluster, and eventually external when properly set (for example using ALB as ingress).

[1] https://kubernetes.io/docs/concepts/services-networking/service
[2] https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types

Reply:
Thanks!

I updated the content to state:

> This resource exposes a pod from a targeted edge worker node to your Local Zone network.